Fe8/unit block editing

Teraspark opened the pull request.

Teraspark committed:
Added missing unit block to chapter 8

Teraspark committed:
Force the messenger to fly in chapter 9 (Eirika)

Teraspark committed:
remove weapon lock tag for Eirika Lord classes

although the Eirika lord classes are able to use the myrmidon weapons, there seems to be an issue with auto leveling not giving them the weapon rank to use them.
The tag will be removed until somebody can take a look at the auto leveling routine.

Teraspark committed:
Valter must fight

Valter has been causing issues with his cutscene battles because he sometimes spawns without weapons.
Now he should be forced to always have a weapon in his inventory

Teraspark committed:
remove commas

Teraspark committed:
Merge branch 'fe8/unit-block-editing' of https://github.com/Teraspark/Archipelago into fe8/unit-block-editing

Teraspark committed:
Merge remote-tracking branch 'upstream/fe8/dev' into fe8/unit-block-editing

CT075 commented on worlds/fe8/data/characters.json:
```diff
@@ -1,65 +1,65 @@
+    "Gerik": {"ids": [20], "tags": []},
+    "Tethys": {"ids": [21], "tags": []},
+    "Marisa": {"ids": [22], "tags": []},
+    "Saleh": {"ids": [23], "tags": []},
```
Is there a reason we dropped `must_fight` here?

Teraspark replied:
I don't think that one really does anything? He should already set to `must_fight` in every significant unit block.
If you want to remove, it shouldn't cause any problems.

CT075 commented on worlds/fe8/data/chapter_unit_blocks.json:
```diff
@@ -162,8 +163,12 @@
+            "name": "MessengerScene2",
+            "base": 9143368,
+            "count": 1,
+            "logic": {"0": {"must_fly": true}}
```
This is actually Tana, so this `must_fly` won't work. Nudging her start position to `[0,5]` instead is probably best.

Teraspark committed:
nudge chapter 9 (Eirika) Messenger

replace "must_fly" with "nudges"

CT075 committed:
nudge starting pirate in c9eir

Signed-off-by: Cameron Wong <cam@camdar.io>

CT075 committed:
fix typo

Signed-off-by: Cameron Wong <cam@camdar.io>

Teraspark committed:
Add "no_flying" tags to Chapter 17

Ban certain units from getting flying classes in Chapter 17.
Tis is to prevent the green units from being killed on turn 1.

CT075 committed:
make cormag's cutscene bros also fly

Signed-off-by: Cameron Wong <cam@camdar.io>
